maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/util/FailsafeSummaryXmlUtils.java

```diff
@@ -84,12 +85,14 @@ public static RunResult toRunResult(File failsafeSummaryXml) throws Exception {
             String skipped = xpath.evaluate("/failsafe-summary/skipped", root);
             String failureMessage = xpath.evaluate("/failsafe-summary/failureMessage", root);
             String timeout = xpath.evaluate("/failsafe-summary/@timeout", root);
+            String flakes = xpath.evaluate("/failsafe-summary/flakes", root);
 
             return new RunResult(
                     parseInt(completed),
                     parseInt(errors),
                     parseInt(failures),
                     parseInt(skipped),
+                    isBlank(flakes) ? 0 : parseInt(flakes),
```

Comment:
nit: do you reckon it's worth adding a comment why we are treating this differently, i.e. because of backward compatibility?

Reply:
I agree here, a comment with a remove/replace to `parseInt()` in the future.